File: android/app/src/main/java/jagtheora/theora/TheoraComment.java
package jagtheora.theora;

import jagtheora.misc.SimplePeer;

public final class TheoraComment extends SimplePeer {

    public TheoraComment() {
        this.init();
        if (this.b()) {
            throw new IllegalStateException();
        }
    }

    protected void init() {}

    protected void clear() {}
}

File: src/jvmMain/kotlin/jagtheora/theora/TheoraComment.kt
package jagtheora.theora

import jagtheora.misc.SimplePeer

actual class TheoraComment : SimplePeer() {
    init {
        init()
        check(!this.b())
    }

    actual external override fun clear()

    companion object {
        /*protected*/ actual external fun init()
    }
}
